sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="Azure.Storage.Blobs" />
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
+    <PackageReference Include="Newtonsoft.Json" VersionOverride="13.0.1"/>
```

Comment:
This doesn't seem correct. Azure.* libraries should not be depending on Newtonsoft.

Reply:
Agree.  There's no reason that this package should be here.

Reply:
This package is here purely because the runtime resolution ends up picking up an older version of the library so this is forcing the correct version resolution. Let me know if there is another way to do it.

Reply:
There is no use of anything JSON in this package.   There's nothing that should be picking up any version of Newtonsoft.

Reply:
I was purely going by the closure\build errors\warning to guide me but let me look closure. But I agree we should not need the override for sure here.